src/Genre.elm

```diff
@@ -45,3 +38,23 @@ fromQueryString location =
         |> String.split "+"
         |> List.filter (\g -> not (String.isEmpty g))
         |> fromFlatList
+
+
+{-| Initializes a multiselect with an available and selected list
+-}
+multiSelectModel : Set Genre -> Set Genre -> Multiselect.Model
+multiSelectModel available selected =
+    available
+        |> Set.toList
+        |> (\m -> Multiselect.initModel m "genrePicker")
+        |> (\m -> { m | selected = Set.toList selected })
+
+
+selectionView : (Multiselect.Msg -> msg) -> Multiselect.Model -> Html msg
```

Comment:
To move `selectionView` into this file, we let it take a function that converts a `Multiselect.Msg` to a `msg` type of the caller's choice.  Then in `Main.elm` we call it, passing the `MultiselecteEvent` constructor, which has the type `(Multiselect.Msg -> Main.Msg)`

Reply:
I could have also let it return `Html Multiselect.Msg` and passed the result to `Html.map MultiselectEvent` in `Main.elm` I'm about 50/50 on which is a better approach.

Reply:
My only feedback here is that you've named a function `msg` below, where it's actually a converting function.

Reply:
Yeah, that's a bit unclear. I'll see if I can come up with something better.

Reply:
Looked around and found this example for a similar use case:
https://github.com/rtfeldman/elm-spa-example/blob/master/src/Views/User/Follow.elm#L25

The message generator function is named in a `verby` way there to indicate it's intended to trigger an action, which seemed a resonable approch, so I went with `updateSelection`